_posts/2024-08-12-AI-chatting.md

```diff
@@ -0,0 +1,95 @@
+
+I want to introduce you to the new AI features in JabRef.
+
+- AI can generate for you a summary of a research paper
+- You can also chat with papers using a smart AI assistant
+
+## AI summary tab
+
+We have made a new entry editor tab: "AI Summary", where AI will generate for you a quick overview of the paper.
+
+![AI summary tab screenshot](../img/AiSummary.png)
+
+The AI will mention for you main objectives of the research, methods used, key findings, and conclusions.
+
+## AI chat tab
+
+The next new entry editor tab is "AI chat", where all the question and answering (Q&A) happens.
+
+![AI chat tab screenshot](../img/AiChat.png)
+
+In this window you can see the following elements:
+
+- Chat history with your messages
+- Prompt for sending messages
+- A button for clearing the chat history (just in case)
+
+Let's try it out on a paper "Cooper, K., Donovan, J., Waterhouse, A., & Williamson, G. (2007). Cocoa and health: a decade of research. *British Journal of Nutrition*, 99(1), 1–11. doi:[10.1017/s0007114507795296](https://doi.org/10.1017/s0007114507795296)"
+
+Let's ask about the chocolate.
+
+![AI first question and answer](../img/AiQuestion1.png)
+
+Correct! The authors talk a lot about chocolate components, and how it can be a powerfull antioxidant.
+Wow! AI has given so much infromation, and it's true: chocolate really contains polyphenols and fatty acids.
+
+I was wondering, how much chocolate should a human eat in a day? Let's dive in this topic.
+
+![AI second question and answer](../img/AiQuestion2.png)
+
+*Only 40 grams?* Well, our health is our wealth. And AI helped us to uncover this information!
+
+## How does this work?
+
+In the background, JabRef analyses the linked PDF files of library entries. The information used after the indexing is then supplied to the AI, which, to be precise, in our case is a Large Language Model (LLM).
+The LLM is currently not stored on your computer. Instead, we have many integrations with AI providers (OpenAI, Mistral AI, Hugging Face), so you can choose the one you like the most.
+These AI providers are available only remotely via the internet. In short: we send chunks of text to AI service and then receive processed responses. In order to use it you need to configure JabRef to use your API key.
+
+## Which AI provider should I use?
+
+We recomend you chosing [OpenAI](https://platform.openai.com/docs/models).
+
+For Mistral AI you need to make a subscription, while for OpenAI you can send money one time.
+
+Hugging Face gives you access to numerous count of models for free.
+
+However, it will take a very long time for Hugging Face to find a free computer resources for you, and the response time will be also long.
+
+## How to get an API key and what should I do after getting one?
+
+Please head to the [AI user documentation](https://docs.jabref.org/ai/ai-providers-and-api-keys) to learn about how to receive a key and where to enter it in the preferences.
```

Comment:
The section contains one sentence only. Remove heading and add following sentence before:

In order to use any service, you always need an API key.